src/christinaGerdzhikova/week03/ReverseNegativeNum.java

```diff
@@ -0,0 +1,30 @@
+package christinaGerdzhikova.week03;
+
+import java.util.Scanner;
+
+public class ReverseNegativeNum {
+
+    public static int reverseNegative(int num){
+
+        if (num > 0){
+            System.out.print("Invalid number: ");
```

Comment:
I'm sorry I'm being so nitpicky!! This could potentially just print "Invalid number: " and not print the number itself at the end. 

The rest of this class is good!

Reply:
![Screenshot 2023-06-04 160914](https://github.com/AndrewSloth/Nova_JavaCodingTasks/assets/97245432/8085c0ff-ff75-4f24-a725-1827216541cf)

the invalid number is printing as well, i attached a screenshot